cibuild.sh

```diff
@@ -7,14 +7,19 @@ usage()
     echo "usage: cibuild.sh [options]"
     echo ""
     echo "Options"
-    echo "  --debug     Build Debug (default)"
-    echo "  --release   Build Release"
-    echo "  --nocache   Force download of toolsets"
-
+    echo "  --debug               Build Debug (default)"
+    echo "  --release             Build Release"
+    echo "  --skiptests           Do not run tests"
+    echo "  --skipcrossgen        Do not crossgen the bootstrapped compiler"
+    echo "  --skipcommitprinting  Do not print commit information"
```

Comment:
If you would rather me just move the commit printing out of here and into the CI definition, I'm happy to do that.  I didn't really understand the point of doing it here (shouldn't the Jenkins logs already provide the information of the SHA you are building?)

This causes problems both on RHEL (where there git is older and doesn't understand `--no-patch` and will be problematic when the sources are not in a git repository).